### PR TITLE
Add horizontal VRM broker scaling

### PR DESCRIPTION
**Pull request by thlassche**
Makes sure the dbus-mqtt client connects to one of the 64
VRM brokers, based on the site id of the client

See: https://github.com/victronenergy/venus/issues/543

@wiebeytec I added a slightly different implementation than we first suggested, because it was convenient for me to just use `ord()` (which is available in most languages) please let me know if you think that will work

**ToDo:**
- [ ] Include this in the submodule of https://github.com/victronenergy/dbus-mqtt
- [ ] Include this in the submodule of https://github.com/victronenergy/mqtt-rpc. Both mqtt projects need to have the same version of the bridge registrator, otherwise they overwrite each other.
- [x] Test the implementations as soon as there are records for the 128 brokers

### CODE REVIEW

File: mosquitto_bridge_registrator.py [hunk 1]
```diff
@@ -81,6 +80,18 @@ def __init__(self, system_id, restart_mosquitto=True):
...
 		self._requests_log_level = logging.getLogger("requests").getEffectiveLevel()
 
+	def _get_vrm_broker_url(self):
+		"""To allow scaling, the VRM broker URL is generated based on the system identifier
+		The function returns a numbered broker URL between 0 and VrmNumberOfBrokers, which makes sure
+		that broker connections are distributed equally between all VRM brokers
+		"""
+		sum = 0
+		for character in self._system_id:
+			sum += ord(character)
```
Comment (wiebeytec) [inline mosquitto_bridge_registrator.py]: Should we be worried about the assumed encoding? This code assumes the source is unicode. If you do this on another platform, it happens to work for ASCII as long there are no special chars, but it's somewhat undefined. And I hope we don't ever see EBCDIC...

Also, we should do a `lower()` and `strip()` for safety.

File: mosquitto_bridge_registrator.py [hunk 2]
```diff
@@ -6,10 +6,9 @@
 import subprocess
 import traceback
 from ve_utils import exit_on_error
-
+VrmNumberOfBrokers = 64
```
Comment (wiebeytec) [inline mosquitto_bridge_registrator.py]: Shall we make it 128 instead? Is more clicking for me, but it's safer.

**Comment by wiebeytec** [comment]
I added a TODO, about the mosquitto_bridge_registrator being used by two projects, and they need to be the same.

**Comment by wiebeytec** [comment]
The DNS records have been made, mqtt0.victronenergy.com - mqtt127.victronenergy.com.

EuroDNS allows you to download the zone file, edit it, and re-upload, so it was easy.

File: mosquitto_bridge_registrator.py (hunk 1 above)
Comment (thlassche) [reply mosquitto_bridge_registrator.py]: Done!

File: mosquitto_bridge_registrator.py (hunk 2 above)
Comment (thlassche) [reply mosquitto_bridge_registrator.py]: Done!

**Comment by thlassche** [comment]
@mpvader I'm not authorized to merge this, but I've wrapped it up and squashed the commits. Can you merge it? Then I'll update the submodules

**Pull request merged**